### ISSUE TITLE
BackStage Back Button doesn't have an accessibility text.

### ISSUE THREAD

**Issue by robertmuehsig** [fluentribbon/Fluent.Ribbon#1125]
If you focus the button and have the Narrator/nvda/any other screenreader open, nothing will be read:

![image](https://user-images.githubusercontent.com/756703/232986728-841079dd-f33d-4736-8444-89465629fae7.png)

---
### Environment

- Fluent.Ribbon __v9__ (and current development)
- Windows __10/11__ (doesn't matter, but the Narrator is quite "broken" on Windows 11 & WPF)
- .NET Framework __?__ (just started the Showcase app).

**Comment by robertmuehsig** [comment]
I retested it with the most recent develop branch (version 10) and at least the Windows Narrator from Windows 11 & JAWS don't read the button.

The AutomationPeers are set, but it is not working as expected:
![image](https://github.com/fluentribbon/Fluent.Ribbon/assets/756703/78e21ad4-c718-41f1-b5ab-57031b4f75b7)

**Comment by robertmuehsig** [comment]
Just tested it with the most recent released version (and tested the develop branch) and it still seems broken - at least on Windows 11 with the Narrator and NVDA nothing happens when I focus the close backstage button.

I tried to fix it myself, but even when I replace the `backbutton` with a very simple button like this nothing happens:

```
                    <Button 
                            Content="Foobar"
                            Grid.Row="0"
                            Padding="22 7 0 7"
                            VerticalAlignment="Top"
                            Background="Yellow"
                            AutomationProperties.Name="Test"
                            >
                        <Button.Margin>
                            <MultiBinding Converter="{x:Static converters:StaticConverters.ThicknessConverter}">
                                <Binding Source="0" />
                                <Binding FallbackValue="0"
                                         Path="TitleBar.ActualHeight"
                                         RelativeSource="{RelativeSource FindAncestor, AncestorType={x:Type Fluent:IRibbonWindow}}" />
                                <Binding Source="0" />
                                <Binding Source="0" />
                            </MultiBinding>
                        </Button.Margin>
                    </Button>
```
It seems that something is blocking the narrator.

Any idea? Sorry to bother you.
